packages/myst-cli/src/process/mdast.ts

```diff
@@ -39,7 +40,7 @@ import { VFile } from 'vfile';
 import { processPageFrontmatter, updateFileInfoFromFrontmatter } from '../frontmatter.js';
 import { selectors } from '../store/index.js';
 import type { ISession } from '../session/types.js';
-import { castSession } from '../session/cache.js';
+import { castSession, writeToCache } from '../session/cache.js';
```

Comment:
Just following up on the `lint` suggestion.

```suggestion
import { castSession } from '../session/cache.js';
```